docs/ocis/development/testing.md

```diff
@@ -589,6 +589,132 @@ The sample `fontsMap.json` file is located in `tests/config/drone/fontsMap.json`
+
+   # Update template values
+   sed -i '/{{- $_ := set .scope "appNameAuthMachine" "authmachine" -}}/a \
+            {{- $_ := set .scope "appNameAuthBasic" "authbasic" -}}' \
+            ./charts/ocis/templates/_common/_tplvalues.tpl
+   ```
+
+2. Install Charts Using k3d
+
+   ```bash
+   k3d cluster create drone \
+       --api-port ocis-server:33199 \
+       -p '80:80@loadbalancer' \
+       -p '443:443@loadbalancer' \
+       --k3s-arg '--tls-san=k3d@server:*' \
+       --k3s-arg '--disable=metrics-server@server:*'
+
+   k3d kubeconfig get drone > kubeconfig.yaml
+
+   chmod 0600 kubeconfig.yaml
+
+   kubectl create configmap coredns-custom \
+       --namespace kube-system \
+       --from-literal='rewritehost.override=rewrite name exact ocis-server host.k3d.internal'
+
+   kubectl -n kube-system rollout restart deployment coredns
+
+   make helm-install-atomic
+   ```
+
+3. Run Test\
+   To run test you should set `K8S` to true:
+   ```bash
+   cd  <path-to-ocis-repo>
+
+   # build and run ocis wrapper
+   make -C ./tests/ociswrapper
+   ./tests/ociswrapper/bin/ociswrapper serve \
+     --url https://ocis-server \
+     --admin-username admin \
+     --admin-password admin \
+     --skip-ocis-run
+
+   K8S=true TEST_SERVER_URL="https://ocis-server" \
+     make test-acceptance-api BEHAT_FEATURE=tests/acceptance/features/apiArchiver/downloadById.feature
+   ```
+
+4. Run Test With ociswrapper
+   ```bash
+   ociswrapper serve \
+     --url https://ocis-server \
+     --admin-username admin \
+     --admin-password admin \
+     --skip-ocis-run
+   ```
+
+5. Run Antivirus Test\
+   To run the clamav service, run these commands deploying the helm chart:
+   ```bash
+   cp -r config/drone/k8s/clamav charts/ocis/templates/
+
+   sed -i 's/{{ *\.Values\.features\.virusscan\.infectedFileHandling *| *quote *}}/"delete"/' \
+           ocis/templates/antivirus/deployment.yaml
+
+   sed -i 's/{{ *\.Values\.features\.virusscan\.infectedFileHandling *| *quote *}}/"delete"/' \
+           ocis/templates/antivirus/deployment.yaml
+
+   sed -i '/name: ANTIVIRUS_SCANNER_TYPE/{n;s/value: *"icap"/value: "clamav"/}' \
+           charts/ocis/templates/antivirus/deployment.yaml
+
+   sed -i '/- name: ANTIVIRUS_SCANNER_TYPE/i \
+            - name: ANTIVIRUS_CLAMAV_SOCKET\n \
+            value: "tcp://clamav:3310"' \
+            charts/ocis/templates/antivirus/deployment.yaml
+   ```
+
+5. Run Email Test\
+   To run the email service, run these commands deploying the helm chart:
+   ```bash
+   cp -r <path-to-ocis-repo>/tests/config/drone/k8s/mailpit charts/ocis/templates/
+   ```
+   After ocis is deployed:
+   ```bash
+   kubectl port-forward svc/mailpit $EMAIL_PORT:8025 -n ocis
+   ```
+
+6. Run auth-app Test\
+   To run the `auth-app` service, run these commands deploying the helm chart:
+   ```bash
+   cp -r <path-to-ocis-repo>/tests/config/drone/k8s/mailpit charts/ocis/templates/
```

Comment:
The auth-app test has the same copy command as in point 5 - email test. Is this correct?